specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -64,6 +64,325 @@
         }
       }
     },
+    "/docs": {
+      "get": {
+        "tags": [
+          "Search"
+        ],
+        "operationId": "DocumentsProxy_SearchGet",
+        "x-ms-examples": {
+          "SearchIndexGetSearch": { "$ref": "./examples/SearchIndexGetSearch.json" }
+        },        
+        "description": "Searches for documents in the Azure Search index.",
+        "parameters": [
+          {
+            "name": "search",
+            "in": "query",
+            "type": "string",
+            "description": "A full-text search query expression; Use null or \"*\" to match all documents.",
```

Comment:
Is it possible to put an `externalDocs` section here? It might be useful to link to the [Lucene syntax article](https://docs.microsoft.com/rest/api/searchservice/lucene-query-syntax-in-azure-search).

Reply:
Thanks- Fixed. I will do another scan on 'null' 

---
In reply to: [208097730](https://github.com/Azure/azure-rest-api-specs/pull/3578#discussion_r208097730) [](ancestors = 208097730)

Reply:
`externalDocs` is not allowed in description of query parameters. ([https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md)) 

---
In reply to: [208101001](https://github.com/Azure/azure-rest-api-specs/pull/3578#discussion_r208101001) [](ancestors = 208101001)